model/src/test/kotlin/utils/FindingsMatcherTest.kt

```diff
@@ -192,14 +194,26 @@ class FindingsMatcherTest : WordSpec() {
 
                 val result = matcher.match(licenseFindings, copyrightFindings)
 
-                result.size shouldBe 2
                 result.getFindings("license nearby")
                     .copyrights.map { it.statement } shouldContainExactlyInAnyOrder listOf(
-                        "statement2",
-                        "statement3"
-                    )
+                    "statement2",
```

Comment:
Sorry for not spotting this earlier, but this seems to be an unintended formatting change.

Reply:
Done.